js/injected.js

```diff
@@ -36,13 +36,15 @@ function isClerk(resourceList){
   return false;
 }
 
+let startTime = performance.now();
```

Comment:
This looks like it was meant for debugging, im guessing to try to gain insight into how the resource load timing within a given tab, interacts with the signal sent to the popup.js script.

Should be removed for a production merge.